docs/java.md

```diff
@@ -0,0 +1,17 @@
+# Java
+
+This doc outlines how Java is packaged within Wolfi, and how to determine which package is right for your needs.
+
+Each version of Java provided by Wolfi consists of the following packages:
+
+- `openjdk-#-jre`: the Java Runtime Environment (JRE)
+- `openjdk-#`: the Java Development Kit (JDK) and the JRE
```

Comment:
JAVA_HOME only works for use-cases where the jdk isn't being used directly.  I don't think we should present ..default-jvm as a compatibility thing. I'd list it front and center as the third Java package available for each version, and describe how it makes it the default JVM and connects the /usr/bin/symlinks to your JDK of choice.